src/ResponseTagger.php

```diff
@@ -36,10 +45,20 @@ class ResponseTagger
      * Constructor
      *
      * @param TagsInterface $client
+     * @param array         $options supported options:
+     *                               - strict (bool) Default: false. If set to true, throws exception when adding empty tags.
      */
-    public function __construct(TagsInterface $client)
+    public function __construct(TagsInterface $client, array $options = array())
     {
         $this->client = $client;
+
+        $resolver = new OptionsResolver();
+
+        $resolver->setDefaults(array(
+            'strict' => false,
+        ));
```

Comment:
hmz, this causes the symfony 2.3 build to break… 
``setAllowedTypes`` was refactored in version 2.6 and i can't seem to find a way around it which supports both the 2.3 and the >= 2.6 ``OptionsResolver``